### PR TITLE
i18n(es): update the `astro-components.md` page.

### PR DESCRIPTION
**Pull request by diegognt**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description
Update the translated content on the `/es/.../astro-components.md` to the latest.

### CODE REVIEW

File: src/pages/es/core-concepts/astro-components.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ import Button from './Button.astro';
 ```
 
 
-### Script del componente
+### Script de un componente
```
Comment (delucis) [inline src/pages/es/core-concepts/astro-components.md:44]: Looks like a few other pages were linking to this section using the `#script-del-componente` fragment. Could you update those pages to use `#script-de-un-componente`?

Comment (diegognt) [reply src/pages/es/core-concepts/astro-components.md:44]: Will do, thanks.

File: src/pages/es/core-concepts/astro-components.md [hunk 2]
```diff
@@ -492,6 +492,18 @@ Astro detecta los módulos JavaScript importados del lado del cliente y luego co
 </script>
 ```
 
+## Componentes HTML
+
+Astro soporta importar y usar archivos `.html` como componentes o colocarlos dentro del subdirectorio `src/pages`. Es posible que quieras usar componentes HTML si estás reusando código de un sitio existente construido sin usar freamwors, o si quieres aseguarte que tu componente no tiene características dinámicas.
```
Comment (kevinzunigacuellar) [inline src/pages/es/core-concepts/astro-components.md:497]:
```suggestion
Astro soporta importar y usar archivos `.html` como componentes o colocarlos dentro del subdirectorio `src/pages`. Es posible que quieras usar componentes HTML si estás reusando código de un sitio existente construido sin usar frameworks, o si quieres aseguarte que tu componente no tenga características dinámicas.
```

File: src/pages/es/core-concepts/astro-components.md [hunk 3]
```diff
@@ -492,6 +492,18 @@ Astro detecta los módulos JavaScript importados del lado del cliente y luego co
...
+
+Los componentes HTML solo deben contener HTML válido, y por lo tanto le faltarán características claves de los componentes de Astro.
+- Ellos no soportan el frontmatter, importaciones del lado del servidor, o expresiones dinámicas.
+- Cualquier etiqueta `<script>` quedan sin agrupar, son tratados como si tuvieran `in:inline`
```
Comment (kevinzunigacuellar) [inline src/pages/es/core-concepts/astro-components.md:501]:
```suggestion
- Cualquier etiqueta `<script>` no se empaquetará,y será tratado como si tuvieran `in:inline`
```

**Review by kevinzunigacuellar — COMMENTED** [review]
Gracias por la PR 🎉. Deje algunos comentarios para que suene más natural, siéntete libre de cambiarlos, borrarlos o combinarlos. Mandame un ping por aqui o discord cuando este lista para merge.

File: src/pages/es/core-concepts/astro-components.md [hunk 4]
```diff
@@ -492,6 +492,18 @@ Astro detecta los módulos JavaScript importados del lado del cliente y luego co
...
+- Ellos solo [pueden referenciar recursos que están en la carpeta `public/`](/es/guides/images/#public).
+
+:::note
+Un [elemento `<slot/>`](/es/core-concepts/astro-components/#slots) dentro de un componente HTML trabajar como lo haría en un componente de Astro. En cambio, para poder usar el elemento [Componente Web HTML Slot](https://developer.mozilla.org/es/docs/Web/HTML/Element/slot) añade `is:inline` al elemento `slot`.
```
[the remark this answers is not in the document]
Comment (diegognt) [reply src/pages/es/core-concepts/astro-components.md:505]: Makes sense, update applied.

File: src/pages/es/core-concepts/astro-components.md [hunk 5]
```diff
@@ -492,6 +492,18 @@ Astro detecta los módulos JavaScript importados del lado del cliente y luego co
...
 ```
 
+## Componentes HTML
+
+Astro soporta importar y usar archivos `.html` como componentes o colocarlos dentro del subdirectorio `src/pages`. Es posible que quieras usar componentes HTML si estás reusando código de un sitio existente construido sin usar freamwors, o si quieres aseguarte que tu componente no tiene características dinámicas.
+
+Los componentes HTML solo deben contener HTML válido, y por lo tanto le faltarán características claves de los componentes de Astro.
+- Ellos no soportan el frontmatter, importaciones del lado del servidor, o expresiones dinámicas.
+- Cualquier etiqueta `<script>` quedan sin agrupar, son tratados como si tuvieran `in:inline`
+- Ellos solo [pueden referenciar recursos que están en la carpeta `public/`](/es/guides/images/#public).
```
[the remark this answers is not in the document]
Comment (diegognt) [reply src/pages/es/core-concepts/astro-components.md:502]: Makes sense, update applied.

File: src/pages/es/core-concepts/astro-components.md (hunk 3 above)
Comment (diegognt) [reply src/pages/es/core-concepts/astro-components.md:501]: Makes sense, update applied.

File: src/pages/es/core-concepts/astro-components.md [hunk 6]
```diff
@@ -492,6 +492,18 @@ Astro detecta los módulos JavaScript importados del lado del cliente y luego co
 </script>
 ```
 
+## Componentes HTML
+
+Astro soporta importar y usar archivos `.html` como componentes o colocarlos dentro del subdirectorio `src/pages`. Es posible que quieras usar componentes HTML si estás reusando código de un sitio existente construido sin usar freamwors, o si quieres aseguarte que tu componente no tiene características dinámicas.
+
+Los componentes HTML solo deben contener HTML válido, y por lo tanto le faltarán características claves de los componentes de Astro.
```
[the remark this answers is not in the document]
Comment (diegognt) [reply src/pages/es/core-concepts/astro-components.md:499]: Makes sense, update applied.

File: src/pages/es/core-concepts/astro-components.md (hunk 2 above)
Comment (diegognt) [reply src/pages/es/core-concepts/astro-components.md:497]: Makes sense, update applied.

**Pull request merged**